erpnext/public/js/controllers/transaction.js

```
@@ -1972,11 +1972,11 @@ erpnext.TransactionController = class TransactionController extends erpnext.taxe
 			});
 		}
 
-		if (this.frm.doc.scrap_items && this.frm.doc.scrap_items.length > 0) {
-			this.frm.set_currency_labels(["rate", "amount"], this.frm.doc.currency, "scrap_items");
-			this.frm.set_currency_labels(["base_rate", "base_amount"], company_currency, "scrap_items");
+		if (this.frm.doc.secondary_items && this.frm.doc.secondary_items.length > 0) {
+			this.frm.set_currency_labels(["rate", "amount"], this.frm.doc.currency, "secondary_items");
+			this.frm.set_currency_labels(["base_rate", "base_amount"], company_currency, "secondary_items");
 
-			var item_grid = this.frm.fields_dict["scrap_items"].grid;
+			var item_grid = this.frm.fields_dict["secondary_items"].grid;
```

Biome (2.4.4)

[error] Shouldn't redeclare 'item_grid'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)